blivet/formats/stratis.py

```diff
@@ -0,0 +1,81 @@
+# stratis.py
+# Device format classes for anaconda's storage configuration module.
+#
+# Copyright (C) 2020  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Vojtech Trefny <vtrefny@redhat.com>
+#
+
+from ..storage_log import log_method_call
+from ..i18n import N_
+from ..size import Size
+from . import DeviceFormat, register_device_format
+
+import logging
+log = logging.getLogger("blivet")
+
+
+class StratisBlockdev(DeviceFormat):
+    """ A Stratis block device """
+
+    _type = "stratis"
+    _name = N_("Stratis block device")
+    _udev_types = ["stratis"]
+    _formattable = False                 # can be formatted
+    _supported = True                    # is supported
+    _linux_native = True                 # for clearpart
+    _min_size = Size("1 MiB")            # FIXME
```

Comment:
We require 1 GiB for our devices.

Reply:
Thanks, I've updated the `_min_size` value.